packages/components/src/internal/components/search/FindSamplesByIdsPageBase.tsx

```diff
@@ -130,19 +103,6 @@ export const FindSamplesByIdsTabbedGridPanelImpl: FC<FindSamplesByIdsTabProps> =
                 }}
                 user={user}
```

Comment:
this usage should now set `showLabelOption` true right? otherwise, the Find By Ids page now no longer shows the print labels in the export menu.